README.md

```diff
@@ -50,16 +50,11 @@ helmc install workflow-dev
 
 ### Postgresql
 
-Postgresql can be installed via `homebrew`:
-
-```
-brew install postgresql
-```
-
-Or via your package manager. For example, on Debian Jessie:
-
+The application and tests use PostgreSQL. To start a local instance via Docker, run `make postgres` and set the following in your shell:
 ```
-apt-get install postgresql libpq-dev
+  export PGHOST=`docker-machine ip $$(docker-machine active) 2>/dev/null || echo 127.0.0.1`
```

Comment:
Looks like you copied this directly from the Makefile. In that case, the `$$(docker-machine active)` gets correctly expanded to the active docker-machine's IP. However in the case of this addition to the README, there should only be a single `$`, otherwise this errors.